### PR TITLE
17 August Update

### PR DESCRIPTION
**Pull request by gabriel-seow**

### CODE REVIEW

File: src/constants/travel.yaml [hunk 1]
```diff
@@ -35,6 +35,11 @@ countries:
       inbound_restricted:
         - PK
       outbound: partial
+    travel_ideas:
+      - alt: 'How safe is it to travel to Dubai'
+        href: 'https://www.tourhero.com/en/magazine/destinations/asia/is-it-safe-to-travel-to-dubai-what-the-country-is-doing-to-be-covid-19-safe/'
+        src: 'dubai-camels-1900x880.jpg'
+        title: 'Is it safe to travel: Dubai'
```
Comment (jannwong) [inline src/constants/travel.yaml:42]: for this article series, let's use the title `Is it safe to travel to Dubai`. change `:` to `to`

File: src/constants/travel.yaml [hunk 2]
```diff
@@ -102,24 +107,24 @@ countries:
...
+        - 'Entry to Armenia is permitted to anyone who does not originate from any highly-impacted COVID-19 countries, and has not been in one for the past 14 days.](https://covid19.gov.am/en/travel_and_coronavirus)'
       visa_quarantine:
-        - 'Travelers who are permitted to enter Armenia are asked to [complete health questionnaires and self-quarantine or self-monitor for 14 days.](https://am.usembassy.gov/u-s-citizen-services/covid-19-information/)'
+        - 'Travelers who are permitted to enter Armenia are required to [self-isolate for 14 days, although if a negative test result is received within the 14 days then this period may be reduced.](https://www.garda.com/crisis24/news-alerts/368686/armenia-authorities-extend-state-of-emergency-through-september-11-and-ease-other-restrictions-as-of-august-12-update-16)'
         - 'Most visa services are [suspended until further notice.](https://am.usembassy.gov/u-s-citizen-services/covid-19-information/)'
     travel:
       domestic: 'yes'
-      inbound: no
-      outbound: no
```
Comment (jannwong) [inline src/constants/travel.yaml:116]: we removed outbound data but didn't replace it. is there no information?

File: src/constants/travel.yaml [hunk 3]
```diff
@@ -102,24 +107,24 @@ countries:
...
+      inbound: partial
   AO:
+    reopening:
+      '2020-09-08': 'Borders remain closed [through to September 8, 2020](https://ao.usembassy.gov/health-alert-u-s-embassy-luanda-june-1-2020/).'
     sources:
       domestic:
         - 'The government of Angola has [implemented measures](https://ao.usembassy.gov/covid-19-information/) to limit the spread of COVID-19.  Schools and universities are closed until further notice, and sporting events, festivals, and gatherings larger than 50 people have been banned.'
+        - 'Under the state of calamity, existing quarantine measures remain in Luanda province and around the Cazengo municipality (Cuanza Norte province). Travel in and out of Luanda and Cazengo remains prohibited and religious activities in these areas are also suspended. Gatherings of more than ten people are prohibited in public spaces, and [no more than 15 people may gather in private spaces](https://www.garda.com/crisis24/news-alerts/369281/angola-authorities-extend-state-of-calamity-measures-through-september-8-update-10)'
```
Comment (jannwong) [inline src/constants/travel.yaml:125]: there is `ten` and `15`. let's just use numbers, 10 and 15 to be consistent

File: src/constants/travel.yaml [hunk 4]
```diff
@@ -598,7 +602,7 @@ countries:
         - 'Nova Scotia [extends the state of emergency until August 23, 2020](https://www.garda.com/crisis24/news-alerts/367371/canada-authorities-extend-state-of-emergency-in-nova-scotia-until-august-23-update-21).'
       inbound:
         - 'Current ban on most international travelers entering the country has been [extended until July 31, 2020](https://www.garda.com/crisis24/news-alerts/355886/canada-authorities-extend-ban-on-most-foreign-travelers-until-july-31-update-18).'
-        - 'Canadian and US authorities announced on July 15, 2020, that the closure of their mutual border has been [extended until at least August 21, 2020](https://www.garda.com/crisis24/news-alerts/360646/canada-authorities-extend-border-closure-with-us-until-august-21-update-20).'
+        - 'Canadian and US authorities announced on July 15, 2020, that the closure of their mutual border has been [extended until at least September 21, 2020](https://www.garda.com/crisis24/news-alerts/360646/canada-authorities-extend-border-closure-with-us-until-august-21-update-20).'
```
Comment (jannwong) [inline src/constants/travel.yaml:605]: I think the announcement for sept 21 was only made a few days ago. so we should change `july 15` to the new date

File: src/constants/travel.yaml [hunk 5]
```diff
@@ -624,8 +628,12 @@ countries:
         - 'The restrictions imposed on social gatherings, public transport, events, and meetings [have been lifted](https://www.garda.com/crisis24/news-alerts/362081/drc-authorities-lift-covid-19-state-of-emergency-july-22-update-19).'
         - 'Domestic passenger flights and road travel have [resumed nationwide](https://www.garda.com/crisis24/news-alerts/363941/congo-authorities-extend-state-of-health-emergency-until-august-19-update-9).'
       inbound:
-        - 'DRC has [suspended all international passenger flights until at least July 30, 2020](https://www.garda.com/crisis24/news-alerts/358551/congo-state-of-health-emergency-extended-until-july-30-update-8). All domestic passenger flights between Kinshasa and the provinces are suspended. All road and river transport between Kinshasa and the provinces are suspended for passenger traffic.'
+        - 'Authorities reopened the country''s borders, ports and airports, on [August 15, 2020.](https://www.garda.com/crisis24/news-alerts/369551/drc-authorities-reopen-borders-august-15-update-20)'
```
Comment (jannwong) [inline src/constants/travel.yaml:631]: reopen date needs to be updated to Aug 15

File: src/constants/travel.yaml [hunk 6]
```diff
@@ -800,14 +809,14 @@ countries:
       outbound: partial
   CU:
     reopening:
-      '2020-08-15': 'International flights will be [permitted from August 15, 2020](https://www.garda.com/crisis24/news-alerts/364986/cuba-authorities-suspend-flights-through-august-15-update-10).'
+      '2020-08-15': 'International flights will be [permitted from September 1, 2020](https://www.garda.com/crisis24/news-alerts/369231/cuba-authorities-suspend-commercial-flights-through-september-1-update-12).'
```
Comment (jannwong) [inline src/constants/travel.yaml:812]: date should change if it's only permitted from Sept 1

File: src/constants/travel.yaml [hunk 7]
```diff
@@ -1150,6 +1159,11 @@ countries:
       domestic: 'partial'
       inbound: 'partial'
       outbound: 'partial'
+    travel_ideas:
+      - alt: 'How safe is it to travel to Egypt'
+        href: 'https://www.tourhero.com/en/magazine/travel/is-it-safe-to-travel-to-egypt-what-the-country-is-doing-to-be-covid-19-safe/'
+        src: 'is-it-safe-to-travel-to-egypt-pyramids-870x600.jpg'
+        title: 'Is it safe to travel: Egypt'
```
Comment (jannwong) [inline src/constants/travel.yaml:1166]: same change in title

File: src/constants/travel.yaml [hunk 8]
```diff
@@ -1173,7 +1187,7 @@ countries:
...
         - 'In Madrid, public gatherings are [limited to ten people and the use of masks is mandatory](https://www.garda.com/crisis24/news-alerts/364016/spain-additional-covid-19-restrictions-to-be-imposed-in-madrid-from-july-30-update-39) in all public spaces, including streets and terraces.'
-        - 'Clubs, bars, and terraces will be allowed to operate until 1.30 am (local time) and all customers must fill in a register for [contact-tracing purposes in the event of an outbreak](https://www.garda.com/crisis24/news-alerts/364016/spain-additional-covid-19-restrictions-to-be-imposed-in-madrid-from-july-30-update-39).'
+        - 'Spanish authorities [ordered nightclubs to close](https://www.garda.com/crisis24/news-alerts/369116/spain-authorities-order-nightclubs-to-close-august-14-update-40).'
```
Comment (jannwong) [inline src/constants/travel.yaml:1190]: night clubs are 2 words

File: src/constants/travel.yaml [hunk 9]
```diff
@@ -1350,13 +1364,15 @@ countries:
...
       inbound:
-        - '[No entry restrictions. However, from 8 June, 2020, mandatory 14-day quarantine for all travelers will be in effect.](https://www.gov.uk/uk-border-control)'
+        - '[No entry restrictions. However, from June 8, 2020, mandatory 14-day quarantine for all travelers will be in effect.](https://www.gov.uk/uk-border-control)'
       outbound:
         - '[The Foreign and Commonwealth Office (FCO) advises against non-essential international travel.](https://www.gov.uk/guidance/coronavirus-covid-19-travel-corridors)'
       visa_quarantine:
         - 'From June 8, 2020, the UK Government has announced that all passengers arriving in the UK will be required to complete a [mandatory 14-day period of self-isolation.](https://www.heathrow.com/customer-support/faq/coronavirus-covid-19)'
         - 'Travellers from affected countries - including UK nationals - are asked to [provide an address where they will self-isolate for 14 days](https://www.bbc.com/news/explainers-52544307). They can be fined £100 for failing to fill in a form with these details.'
```
Comment (jannwong) [inline src/constants/travel.yaml:1374]: travelers (US spelling)

File: src/constants/travel.yaml [hunk 10]
```diff
@@ -2367,15 +2385,38 @@ countries:
...
       inbound:
         - '[Kuwait has resumed international flights to 20 countries](https://www.garda.com/crisis24/news-alerts/365421/kuwait-international-flights-to-resume-from-august-1-update-25).'
+        - '[Destinations revealed](https://www.arabianbusiness.com/transport/450283-kuwait-to-resume-flights-to-20-destinations-from-august-1#:~:text=Destinations%20revealed%20by%20the%20Directorate,Azerbaijan%2C%20the%20Philippines%20and%20India) by the Directorate General of Civil Aviation (DGCA) include the UAE, Bahrain, Oman, Lebanon, Qatar, Jordan, Egypt, Bosnia and Herzegovina, Sri Lanka, Pakistan, Ethiopia, the United Kingdom, Turkey, Iran, Nepal, Switzerland, Germany, Azerbaijan, the Philippines and India.'
```
Comment (jannwong) [inline src/constants/travel.yaml:2391]: this sentence doesn't really make sense. Destinations revealed for?

I think it's related to the above statement but unclear. So perhaps a better way to phrase it is "Flights resumption for 20 countries include: ..."

File: src/constants/travel.yaml [hunk 11]
```diff
@@ -3128,12 +3185,12 @@ countries:
...
         - 'Mexican and US authorities announced that the closure of their mutual border has been [extended until at least August 21](https://mx.usembassy.gov/travel-restrictions-fact-sheet/).'
       outbound:
-        - 'Land border restrictions with the US, which were introduced due to COVID-19, will be [extended through August 21, 2020](https://www.garda.com/crisis24/news-alerts/355486/mexico-authorities-further-ease-covid-19-restrictions-in-mexico-city-june-29-update-17)'
-        - 'All airports of entry are operating, [possibly on a reduced schedule](https://www.universalweather.com/blog/mexico-coronavirus-covid-19-impact-on-business-aviation/).'
+        - 'Land border restrictions with the US, which were introduced due to COVID-19, will be [extended through September 21, 2020](https://www.garda.com/crisis24/news-alerts/355486/mexico-authorities-further-ease-covid-19-restrictions-in-mexico-city-june-29-update-17)'
+        - 'All nonessential travel including [recreation and tourism remains prohibited](https://www.garda.com/crisis24/news-alerts/368936/mexico-health-authorities-confirm-over-500000-covid-19-cases-august-14-update-22).'
```
Comment (jannwong) [inline src/constants/travel.yaml:3193]: non-essential

File: src/constants/travel.yaml [hunk 12]
```diff
@@ -3181,7 +3238,7 @@ countries:
     sources:
       domestic:
         - 'Walvis Bay (Erongo region), remains under strict movement restrictions where gatherings of more than ten people are prohibited, and [permits will be required to travel to other areas in Namibia](https://www.garda.com/crisis24/news-alerts/353431/namibia-authorities-announce-further-easing-of-some-covid-19-restrictions-june-22-update-7)'
-        - 'Casinos will be permitted to [reopen for pre-booked clients only](https://www.garda.com/crisis24/news-alerts/353431/namibia-authorities-announce-further-easing-of-some-covid-19-restrictions-june-22-update-7). Public gatherings including weddings, funerals, and religious services will be allowed a maximum of 100 guests. Sporting events and activities will also be permitted, subject to limited public gatherings.'
+        - 'Under increased restrictions, mandatory wearing of face masks in public, a limit on public gatherings of ten people, the closure of nightclubs, gambling houses, and casinos, and limitations on the sale of alcohol [will be imposed](https://www.garda.com/crisis24/news-alerts/369286/namibia-authorities-tighten-covid-19-restrictions-from-august-12-28-update-9).'
```
Comment (jannwong) [inline src/constants/travel.yaml:3241]: 10 people
night clubs (2 words)

File: src/constants/travel.yaml (hunk 2 above)
Comment (gabriel-seow) [reply src/constants/travel.yaml:116]: no information on outbound so far!

File: src/constants/travel.yaml (hunk 9 above)
Comment (gabriel-seow) [reply src/constants/travel.yaml:1374]: it's already in US spelling hahaha

File: src/constants/travel.yaml [hunk 13]
```diff
@@ -616,16 +620,20 @@ countries:
         title: 'Newfoundland Getaway: 48 Hours in Gros Morne National Park'
   CD:
     reopening:
-      '2020-07-30': 'DRC has [suspended all international passenger flights until at least July 30, 2020](https://www.garda.com/crisis24/news-alerts/358551/congo-state-of-health-emergency-extended-until-july-30-update-8).'
+      '2020-07-30': 'DRC has [suspended all international passenger flights until at least August 15, 2020](https://www.garda.com/crisis24/news-alerts/358551/congo-state-of-health-emergency-extended-until-july-30-update-8).'
```
Comment (jannwong) [inline src/constants/travel.yaml:623]: reopen date is august 15

**Pull request merged**